### ISSUE TITLE
linux support

### ISSUE THREAD

**Issue by hacknorris-code** [THOMASVGTR/ROOTING-FLASHING-PROCCESS-FOR-XIAOMI-REDMI-GO#1]
add gsi installer for linux (i see only exe) cause right now you denied you description lol

**Comment by THOMASVGTR** [comment]
Linux support is on the way.

**Comment by hacknorris-code** [comment]
ok, thx although i need it pretty quicly :)

**Comment by THOMASVGTR** [comment]
for now, please use windows as I said [Recommended].

**Comment by hacknorris-code** [comment]
i dont have lol

**Comment by THOMASVGTR** [comment]
Okay then, wait sometime.

**Comment by hacknorris-code** [comment]
:(

**Comment by THOMASVGTR** [comment]
Because that tool is not currently ready yet. I couldn't manage my time for this now. But, I will try to finish this. Don't worry. For now, please give a try with windows.

**Comment by hacknorris-code** [comment]
i dont think will work under wine...

**Comment by THOMASVGTR** [comment]
No, Wine & this tool is totally unsupported for now.

**Comment by hacknorris-code** [comment]
yeah and physically i dont have any version of windows neither money to buy it

**Comment by THOMASVGTR** [comment]
you don't need to buy.. it's free. Just download the win10 or win11 iso and install.

**Comment by hacknorris-code** [comment]
disk is full anyway

**Comment by THOMASVGTR** [comment]
wow😅

**Comment by hacknorris-code** [comment]
literally i delete roms after i failed once to be able to download other lol
